flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -162,9 +162,9 @@
 	 * Non-volatile, because only accessed in synchronized scope */
 	private boolean periodicScheduling;
 
-	/** Flag whether a trigger request could not be handled immediately. Non-volatile, because only
-	 * accessed in synchronized scope */
-	private boolean triggerRequestQueued;
+	/** Flag whether periodic triggering is suspended (too many concurrent pending checkpoint).
+	 * Non-volatile, because only accessed in synchronized scope */
+	private boolean periodicTriggeringSuspended;
```

Comment:
Could you rephrase and elaborate the commit message
```
Actually the newcome trigger request will be failed if there are too many pending checkpoints.
It's more like suspending periodic triggering rather than queuing.
And the semantic of queuing will be used in later commit.
```
? I'm struggling to understand the motivation behind this commit?

Reply:
OK let me explain more about the background.
1. I want to introduce a queue to cache those trigger requests that can't be triggered for now. Because there is an asynchronous in-flight trigger. This is described in last commit.
2. I found there is already a queue logic existing, `triggerRequestQueued`.
3. I tried to unify these two queuing concepts. But I found it makes things much more complicated. It's hard to understand or maintain the queuing logic without this background. I believe it's more reasonable to keep both these queuing concepts separately.
4. I renamed the `triggerRequestQueued` to `periodicTriggeringSuspended` to distinguish these two concepts. I think the `triggerRequestQueued` relevant concept is not correct somewhat. It does not really queue the trigger request but fail it and suspend the periodic triggering instead.

Reply:
Ok, thanks for the extra context (this explanation I think also could be added to this commit message). I was also confused/didn't like the previous `triggerRequestQueued` name.

However I'm still missing if this commit changes any behaviour or is it just pure rename/clean up?

Reply:
I have explained a bit more in the response inline. It's not just a pure renaming. To be honest,  after so many changes the codes are not easy to maintain. So I did a bit refactoring when I believe it's necessary.

Reply:
Good suggestion. Will do that later.